Dockerfile

```diff
@@ -3,12 +3,30 @@
 
 FROM concourse/dev
 
+
 # download go modules separately so this doesn't re-run on every change
 WORKDIR /src
 COPY go.mod .
 COPY go.sum .
 RUN grep '^replace' go.mod || go mod download
 
+
+# containerd tooling
+ARG RUNC_VERSION=v1.0.0-rc8
+ARG CNI_VERSION=v0.8.2
+ARG CONTAINERD_VERSION=1.3.0
+
+RUN apt update && apt install -y curl
+
+RUN set -x && \
+	curl -sSL https://github.com/containerd/containerd/releases/download/v$CONTAINERD_VERSION/containerd-$CONTAINERD_VERSION.linux-amd64.tar.gz \
+		| tar -zvxf - -C /usr/local/concourse/bin --strip-components=1 && \
+	curl -sSL https://github.com/opencontainers/runc/releases/download/$RUNC_VERSION/runc.amd64 \ 
+		-o /usr/local/concourse/bin/runc && chmod +x /usr/local/concourse/bin/runc && \
+	curl -sSL https://github.com/containernetworking/plugins/releases/download/$CNI_VERSION/cni-plugins-linux-amd64-$CNI_VERSION.tgz \
+		| tar -zvxf - -C /usr/local/concourse/bin
```

Comment:
this can be removed in a subsequent PR after getting  https://github.com/concourse/ci/pull/204 properly figured out - having this in this PR makes it "complete" though (can be tested with `CONCOURSE_GARDEN_USE_CONTAINERD=true`) from macos/etc